groclient/client.py

```diff
@@ -1300,6 +1300,8 @@ def get_data_points(self, **selections):
             True by default, will include historical regions that are part of your selections
         available_since : string, optional
             Fetch points since last data retrieval where available date is equal to or after this date
+        show_metadata : bool, optional
```

Comment:
@cn1036 do you remember why we didn't show this flag before?

Reply:
I can't recall. this param exists in lib.py since 2020. in my opinion, there is no harm to expose it to the end users.